addons/website/static/src/scss/website.scss

```diff
@@ -1029,18 +1029,19 @@ header {
             // Hack: padding is used by layout option (boxed, etc) so use
             // border here to be able to combine the effect.
             @if $-hamburger-right {
-                border-right: o-website-value('sidebar-width') solid transparent;
+                padding-right: o-website-value('sidebar-width');
             } @else {
                 border-left: o-website-value('sidebar-width') solid transparent;
             }
```

Comment:
It is explained right above the line you modify why border should be used and not padding... and you change it into padding...